include/smrt/Multi_Pin_Subchannel.h

```diff
@@ -49,18 +53,30 @@ class Multi_Pin_Subchannel {
 
   SP_Pin_Subchannel d_pin_subchannel;
 
+  //! coolant temperature in [K] for each channel, of total length given by the
+  //! product of the number of pins by the number of axial cells
+  std::vector<double> pin_temps;
+
+  //! coolant density in [g/cm^3] for each channel, of total length given by the
+  //! product of the number of pins by the number of axial cells
+  std::vector<double> pin_densities;
```

Comment:
For consistency, let's name these `d_pin_temps` and `d_pin_densities`. Later (as a separate PR) we should change the SMRT classes to follow the naming convention in our [style guide](https://enrico-docs.readthedocs.io/en/latest/devguide.html#naming) (i.e., `name_` instead of `d_name`).
```suggestion
  std::vector<double> d_pin_densities;
```

Reply:
Sounds good, that will make this easier to understand